src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -2222,6 +2222,7 @@ object SampleWdl {
         |  String c = b + "MB"
         |  command { ps }
         |  runtime {
+        |    docker: "ubuntu:latest"
```

Comment:
Yes, I changed a few of the tests to use a jesBackend instead of a LocalBackend, because now only supported attributes (for a particular backend) are taken into consideration and validated, and some tests are using the "memory" runtime attribute which is only supported by JES. Because of that the corresponding wdls need to have a docker so the validation doesn't fail.